src/Integration/LocalServices/ProjectSystemHelper.cs

```diff
@@ -67,6 +67,13 @@ public ProjectSystemHelper(IServiceProvider serviceProvider)
             this.serviceProvider = serviceProvider;
         }
 
+        public IVsHierarchy GetFileVsHierarchy(string fileName)
```

Comment:
I debated if it's better to implement this in the Infrastructure.VS project, but:
1. I didn't want to have it reference the DTE
2. I needed `GetIVsHierarchy` method, which also relies on DTE